### PR TITLE
Retail `z_player_lib.c` OK

### PR DESCRIPTION
**Pull request by engineer124**

### CODE REVIEW

File: src/code/z_player_lib.c [hunk 1]
```diff
@@ -1507,6 +1475,8 @@ void Player_PostLimbDrawGameplay(PlayState* play, s32 limbIndex, Gfx** dList, Ve
         if (this->actor.scale.y >= 0.0f) {
             if (!Player_HoldsHookshot(this) && ((hookedActor = this->heldActor) != NULL)) {
                 if (this->stateFlags1 & PLAYER_STATE1_9) {
+                    static Vec3f D_80126128 = { 398.0f, 1419.0f, 244.0f };
+
```
Comment (engineer124) [inline src/code/z_player_lib.c:1479]: This data is needed here for regalloc, and to preserve ordering of data, all the data below needs to be moved in-function

File: src/code/z_player_lib.c [hunk 2]
```diff
@@ -1577,15 +1551,25 @@ void Player_PostLimbDrawGameplay(PlayState* play, s32 limbIndex, Gfx** dList, Ve
...
+            };
+
             Matrix_Get(&this->shieldMf);
             Player_UpdateShieldCollider(play, this, &this->shieldQuad, sRightHandLimbModelShieldQuadVertices);
         }
 
         if (this->actor.scale.y >= 0.0f) {
             if ((this->heldItemAction == PLAYER_IA_HOOKSHOT) || (this->heldItemAction == PLAYER_IA_LONGSHOT)) {
+                static Vec3f D_80126184 = { 100.0f, 1500.0f, 0.0f };
                 Matrix_MultVec3f(&D_80126184, &this->unk_3C8);
```
Comment (fig02) [inline src/code/z_player_lib.c:1569]: all this data being inline makes sense anyway imo :)
line break here

**Review by fig02 — APPROVED** [review]

File: src/code/z_player_lib.c [hunk 3]
```diff
@@ -1536,6 +1506,10 @@ void Player_PostLimbDrawGameplay(PlayState* play, s32 limbIndex, Gfx** dList, Ve
             Matrix_Get(&this->shieldMf);
         } else if ((this->rightHandType == PLAYER_MODELTYPE_RH_BOW_SLINGSHOT) ||
                    (this->rightHandType == PLAYER_MODELTYPE_RH_BOW_SLINGSHOT_2)) {
+            static BowSlingshotStringData sBowSlingshotStringData[] = {
+                { gLinkAdultBowStringDL, { 0.0f, -360.4f, 0.0f } },        // Bow
+                { gLinkChildSlingshotStringDL, { 606.0f, 236.0f, 0.0f } }, // Slingshot
+            };
```
Comment (cadmic) [inline src/code/z_player_lib.c:1512]: sorry, should line breaks go after all of these too? There's a couple more below

**Review by cadmic — APPROVED** [review]

File: src/code/z_player_lib.c (hunk 3 above)
Comment (fig02) [reply src/code/z_player_lib.c:1512]: this case and all the others still have more variables being newly declared under it. the linebreak goes after the declarations are done (even if those declarations are also assigning)

**Pull request merged**